content/v2.0/organizations/members/add-member.md

```diff
@@ -7,25 +7,33 @@ menu:
     name: Add a member
     parent: Manage members
 weight: 201
-draft: true
 ---
 
-Use the InfluxDB user interface (UI) or the `influx` command line interface (CLI)
-to add a member to an organization.
+Use the `influx` command line interface (CLI) to add a member to an organization.
 
-## Add a member to an organization in the InfluxDB UI
+<!-- ## Add a member to an organization in the InfluxDB UI -->
 
-1. Click the **Settings** tab in the navigation bar.
+<!-- 1. Click the **Settings** tab in the navigation bar. -->
 
-    {{< nav-icon "settings" >}}
+<!--     {{< nav-icon "settings" >}} -->
 
-2. Click on the name of an organization, then select the **Members** tab.
+<!-- 2. Click on the name of an organization, then select the **Members** tab. -->
 
-_Complete content coming soon_
+<!-- _Complete content coming soon_ -->
```

Comment:
No need to comment out each line individually. Just comment out the enter section in a single comment.